cmd/swarmctl/service/logs.go

```diff
@@ -23,6 +23,8 @@ var (
 			}
 
 			follow, err := cmd.Flags().GetBool("follow")
+			noStdout, err := cmd.Flags().GetBool("no-stdout")
+			noStderr, err := cmd.Flags().GetBool("no-stderr")
```

Comment:
err check at the bottom covers all 3 cases.

There aren't similar flags in docker, in either service logs or regular container logs.